src/rrt/Tree.hpp

```diff
@@ -52,7 +68,10 @@ class Node {
      */
     const T& state() const { return _state; }
 
+    double* coordinates() { return _vec.data(); }
```

Comment:
Yeah actually double* isn't a great return format. std::vector& would probably be best.